piksi_tools/bootload.py

```diff
@@ -81,7 +81,7 @@ def handshake(self, timeout=None):
       t0 = time.time()
     self.handshake_received = False
     expire = time.time() + 15.0
-    self.link.send(SBP_MSG_RESET, "")
+    self.link(MsgReset())
```

Comment:
nit: Actually, I'm pretty sure there's a `self.link.reset()` that does just this.